### DISCUSSION TITLE
[Feature Request] Video loop for mobile

### DISCUSSION THREAD

**Discussion by joelis10** [immich-app/immich#7623] [Feature Request]
### The feature

One of the biggest (if only) things I miss from Google Photos. It's a little annoying to have to keep pressing play anytime my short timelapse videos end, so a loop button (or even just loop on by default) would be a lovely feature to have. Web can currently loop video through the browser's context menu, but mobile is missing out. I'm also aware that there's another feature request for looping video already, but it was opened nearly a year ago so I wanted to get fresh responses.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
> I'm also aware that there's another feature request for looping video already, but it was opened nearly a year ago so I wanted to get fresh responses.

That's not how it works. There is a huge amount of open feature requests. Most of those have not seen activity in quite a while, but that doesn't mean much.
